src/ios/CDVCapture.m

```diff
@@ -292,6 +295,65 @@ - (CDVPluginResult*)processVideo:(NSString*)moviePath forCallbackId:(NSString*)c
     return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:fileArray];
 }
 
+- (void)showAlertIfAccessProhibeted
+{
```

Comment:
I put `{` in the same place as it's in other functions. So, this is from existing code:
```
- (void)getFormatData:(CDVInvokedUrlCommand*)command
{
```
And it's written like this for every function in that file.
Maybe I missed something?